guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+
+[id="proc_enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+[role="_abstract"]
+A fresh installation of {Project} does not have any Puppet integration configured by default.
+You need to enable the integration as is appropriate for your situation.
+{Project} can be configured to manage and deploy Puppet Server on {ProjectServer} or on {SmartProxy}.
+Additionally, Puppet Server can be deployed externally to {Project} and integrated for reporting, facts and external node classification (ENC).
+
+.Prerequisites
+
+* You installed and configured {Project}.
+
+.Procedure
+
+. Enable Puppet on {ProjectServer} before you enable it on {SmartProxy}s:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-puppet \
+--enable-foreman-cli-puppet \
+--foreman-proxy-puppet true \
+--foreman-proxy-puppetca true \
+--foreman-proxy-content-puppet true \
```

Comment:
This should be hidden behind ifdef and only apply to Katello and its derivatives.

Reply:
The background is that content is a Katello feature. That means any installer option that starts with `--foreman-proxy-content-` is just for Katello. So in this case:
```suggestion
 ifdef::katello,satellite,orcharhino[]
--foreman-proxy-content-puppet true \
 endif::[]
```